CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing to Celestia
+
+Welcome and thank you for contributing to building Celestia, the first modular
+blockchain network.
+
+In this guide, you will find information about how the Celestia team manages the
+`celestiaorg` Github as well as expectations around our engineering standards.
+
+## Github Management
+
+The `celestiaorg` Github uses teams to manage access to the organization and
+its repositories. Members of the Celestia core team and key contributors are
+added to the Github team that aligns with the scope of work they perform. For
+example, those working on the `celestia-node` repository are added to the
+`celestia-node` team.
+
+### Permissions
+
+By default, all teams have `Triage` access to all repositories in the
+`celestiaorg` Github. This allows for anyone to help manage issues and pull
+requests (i.e. adding labels). Teams are given write access to the repositories
+that they are responsible for working on. For example, the `celestia-node` team
+would have write access to the `celestia-node` repository.
+
+### Codeowners
+
+All production repos must use Codeowners. Codeowners are typically the team
+leads and/or engineering leadership members.
+
+### .github
+
+The Celestia team utilizes the organization's `.github` repository. This
+repository is used to store common organization level content like Github
+actions, issue templates, PR templates, etc.
+
+### Repository Settings
+
+The following is a list of key settings that should be enabled on all production
+repositories:
+
+**Enabled:**
+
+- Issues
+- Projects
+- Perserve this repository
+- Allow squash merging
+  - Default to pull request title
+- Always suggest updating pull request branches
+- Allow auto-merge
+- Automatically delete head branches
+
+**Disabled:**
+
+- Allow merge commits
+- Allow rebase merging
+
+## Development
+
+### What the Fork
+
+The default development flow is to fork the repository that you are working on
+in order to submit a PR. If you have write access to a repository, because you
+are a member of that sub team, then you can push your development branches
+directly to the repository.
+
+### ADRs
+
+For architectural changes or improvements, Celestia uses Architecture Decision
+Record (ADRs) to flush out the design scope. These live in the code under
+`docs/adr`. PRs are used to open new ADRs for approval.
```

Comment:
[non-blocking]

> These live in the code under `docs/adr`

I'm on-board with having ADRs in a consistent location across repos but flagging that this isn't the case today:
1. https://github.com/celestiaorg/celestia-app/tree/main/docs/architecture
2. https://github.com/celestiaorg/celestia-core/tree/v0.34.x-celestia/docs/celestia-architecture
3. https://github.com/celestiaorg/rollmint/tree/main/docs/lazy-adr

celestia-node conforms https://github.com/celestiaorg/celestia-node/tree/main/docs/adr but it's a bit difficult for celestia-core to conform because there are already Tendermint ADRs in https://github.com/celestiaorg/celestia-core/tree/v0.34.x-celestia/docs. If we're okay with an exception for celestia-core, (i.e. `docs/celestia-adr`) then I think we can update the non-conformant repos in follow-up PRs.

Reply:
Yup thanks for this. As an fyi once this is merge I had a follow up to do a full organization audit to address discrepancies 👍